spec/controllers/notifications_controller_spec.rb

```diff
@@ -149,17 +149,17 @@
       FactoryGirl.create(:notification, :recipient => alice, :target => @post)
       FactoryGirl.create(:notification, :recipient => alice, :target => @post)
```

Comment:
Is it enough to change the corresponding "it" blocks? Or you mean to update the whole _spec.rb file?